source/code-snippets/usage-examples/replaceOne.js

```diff
@@ -0,0 +1,51 @@
+// ignored first line
+const { MongoClient } = require("mongodb");
+
+// Replace the following with your MongoDB deployment's connection string.
+const uri =
+  "mongodb+srv://<user>:<password>@<cluster-url>?retryWrites=true&w=majority";
+
+const client = new MongoClient(uri);
+
+async function run() {
+  try {
+    await client.connect();
+
+    const database = client.db("sample_mflix");
+    const collection = database.collection("movies");
+
+    // create a filter for a movie to update
```

Comment:
Would it make sense to use the same term (query) as the documentation (on this line and the next)?

Reply:
absolutely. doing this now